docs/blog/posts/2024-12-11-ultralytics-attack-analysis.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Supply-chain attack analysis: Ultralytics"
+description: Analysis of a package targeted by a supply-chain attack to the build and release process
+authors:
+  - sethmlarson
+date: 2024-12-11
+tags:
+  - security
+---
+
+Last week, the Python project “[ultralytics](https://pypi.org/project/ultralytics/)” suffered a supply-chain attack through a compromise of the projects’ GitHub Actions workflows and subsequently its PyPI API token. No security flaw in PyPI was used to execute this attack. Versions 8.3.41, 8.3.42, 8.3.45, and 8.3.46 were affected and have been removed from PyPI.
+
+The attack highlights the importance of securing software forges and the build and publish workflows for open source projects. A [complete set of details is available](https://blog.yossarian.net/2024/12/06/zizmor-ultralytics-injection#conclusions) thanks to the analysis of William Woodruff from Trail of Bits.
```

Comment:
Did you want to link to conclusions directly, or the blog in general, as you link again to conclusions later?